TidepoolOnboardingKitUI/Views/Accessory/AccessibleImage.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AccessibleImage.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 3/22/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+
+struct AccessibleImage: View {
+    private let name: String
+    private let decorative: Bool
+    private let width: CGFloat
+
+    @ScaledMetric private var scalingFactor: CGFloat = 1
```

Comment:
[nit] should this be `let`?

Reply:
`@ScaledMetric` properties must be `var` since SwiftUI will change their value when the system text size changes.